rust/src/index/vector/opq.rs

```diff
@@ -135,6 +137,14 @@ impl OptimizedProductQuantizer {
     }
 }
 
+/// Initialize rotation matrix.
+fn init_rotation(dimension: usize) -> Result<MatrixView> {
+    let mat = MatrixView::random(dimension, dimension);
+    let (u, _, vt) = mat.svd()?;
```

Comment:
here we're using SVD in-place of FAISS's QR thing right?

Reply:
yes